kiwi/repository/zypper.py

```diff
@@ -253,7 +253,11 @@ def import_trusted_keys(self, signing_keys):
         :param list signing_keys: list of the key files to import
         """
         for key in signing_keys:
-            Command.run(['rpm', '--root', self.root_dir, '--import', key])
+            # Including --dbpath flag is a workaround for bsc#1112357
+            Command.run([
+                'rpm', '--root', self.root_dir, '--import',
+                key, '--dbpath', '/var/lib/rpm'
```

Comment:
Could we put the static path into a classmethod in the Defaults space ? I think we should maintain those type of static information all in one place.